src/Search/SearchResult.php

```diff
@@ -119,12 +166,17 @@ public function getLimit(): int
 
     public function getHitsCount(): int
     {
-        return $this->nbHits;
+        return $this->hitsCount;
+    }
+
+    public function count(): int
+    {
+        return $this->hitsCount;
```

Comment:
There are functions with comments and some without. I'm not sure this is important right now but it might impact IntelliSense's relevancy.

Reply:
Yes, IntelliSence is indeed a future project, but not a priority for the MVP here.